sdk/program/src/hash.rs

```diff
@@ -46,6 +46,7 @@ const MAX_BASE58_LEN: usize = 44;
     Pod,
     Zeroable,
 )]
+#[borsh(crate = "borsh")]
```

Comment:
Incidentally, is there a good place to read documentation of `borsh` tags? I looked on their repo and rust docs, but must have missed it.

Reply:
The docs.rs was the best source at https://docs.rs/borsh/latest/borsh/derive.BorshDeserialize.html but I did also have to read the source directly too. This one tripped me up for awhile